build/build.xml

```diff
@@ -217,7 +217,7 @@
     <download-library name="Robot_Motor" version="1.0.2"/>
     <download-library name="RobotIRremote" version="1.0.2"/>
     <download-library name="SpacebrewYun" version="1.0.0"/>
-    <download-library name="Temboo" version="1.1.5"/>
+    <download-library name="Temboo" version="1.1.6"/>
```

Comment:
Temboo is not related. PR is for a specific topic, and commit + commit comments should be for a single purpose.